index.html

```diff
@@ -2,15 +2,11 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=devi1ce-width, initial-scale=1.0" />
     <title>Pull Request Training</title>
   </head>
   <body>
-    <h1>Hello World!</h1>
-    <p>
-      Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto sequi
-      dolorem sit, excepturi pariatur voluptatem qui laudantium voluptatum quam
-      culpa.
-    </p>
+    <h1>Hello John!!</h1>
```

Comment:
Good copywriting. Do you want me to assign u for the copywriting parts of the website?